src/profile_manager/tests/test_views.py

```diff
@@ -516,7 +514,7 @@ def test_profile_update_email(self):
         # Prepare new data for student
         email = f"{self.test_student1.username}@example.com"
 
-        form_data = generate_form_data(model_form=ProfileForm, grad_year=timezone.now().date().year + 2)
+        form_data = generate_form_data(model_form=ProfileForm, custom_profile_field=timezone.now().date().year + 2)
```

Comment:
custom_profile_field is no longer require correct?  So better to leave blank, unless the field itself is being tested/checked.  And if it is, use something like "custom field value" instead of the date expression that will be confusing and has no context any more.  (same with tests below)

Reply:
done